api/internal/core/store/store.go

```diff
@@ -35,8 +35,8 @@ import (
 )
 
 type Interface interface {
-	Get(key string) (interface{}, error)
-	List(input ListInput) (*ListOutput, error)
+	Get(ctx context.Context, key string) (interface{}, error)
+	List(ctx context.Context, input ListInput) (*ListOutput, error)
```

Comment:
`Get` and `List` is work in memory, how to use the context?

Reply:
we don't use it currently, but it's is convenient to pass custom parameters or other data in custom development

Reply:
Keep parameter enough would be better, is here any concret cases which must need context? It will need context only when do some io operations(disk io or network io) as far as I know, store is responsible to query data in memory, so it looks like will never do IO operations.

Reply:
please @nic-chen confirm

Reply:
For example, there may be some business parameters that need to be passed. If we pass these custom parameters one by one, the handler code needs to be modified, and the code will be conflicts  with upstream. If use `ctx` to deliver, user only need to pay attention to the business part.